src/components/Console.js

```diff
@@ -32,6 +33,12 @@ class Console extends Component {
   render() {
     const { console } = this.props;
 
+    const ClearButton = withStyles(() => ({
+      root: {
+        marginTop: '10px',
```

Comment:
Nitpick: could use `theme.spacing(2)` here.
But don't need to make that change now, since we'll be moving stuff around to match the mock soon.

Reply:
Went ahead and changed it since it's simple to do